wallets/connect.go

```diff
@@ -0,0 +1,47 @@
+package wallets
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+)
+
+//  connectWallet connects a new wallet with the given type.
+func connectWallet(c *cli.Context) error {
+	store, ok := c.App.Metadata["store"].(Store)
+	if !ok {
+		return errors.New("failed to get store from context")
+	}
+
+	walletType := c.String("type")
+
+	switch walletType {
+	case WalletTypeAlby:
+		token := c.String("token")
+		if token == "" {
+			return fmt.Errorf("token argument is required for Alby wallets")
+		}
+
+		_, err := ConnectAlbyWallet(store, token)
+		if err != nil {
+			return err
+		}
+
+	case WalletTypeZBD:
+		token := c.String("token")
+		if token == "" {
+			return fmt.Errorf("token argument is required for Alby wallets")
+		}
```

Comment:
is it worth having 2 different swtich cases when both do exactly the same (insert wallet + insert wallet token) but just with dfferent wallet type? We can just pass the type along.

Reply:
Good point I will merge them